app/lib/config/desktop.dart

```diff
@@ -54,6 +54,10 @@ class _DesktopSupportState extends State<DesktopSupport>
     );
     Menu menu = Menu(
       items: [
+        MenuItem(
+          key: 'search',
+          label: 'Search',
+        ),
```

Comment:
I don't know what `desktop.dart` file but I added search page in menu item as there where other Menu where there.

Reply:
Which design it's following for the desktop now? Could you share video preview?

Reply:
Yes sure, updated Desktop Video in PR Description.